src/operator/tensor/matrix_op-inl.h

```diff
@@ -432,7 +419,20 @@ inline bool SliceForwardInferStorageType(const nnvm::NodeAttrs& attrs,
       && (!param.step[0].has_value() || param.step[0].value() == 1)) {
     trivial_step = true;
   }
-  if (!dispatched && in_stype == kDefaultStorage) {
+  if (!dispatched && in_stype == kDefaultStorage && trivial_step) {
```

Comment:
what's the `trivial_step` mean?

Reply:
trivial_step is a flag variable, if trivial_step = false, it will call fallback function.